.travis.yml

```diff
@@ -99,6 +99,7 @@ script:
     make test CTEST_OUTPUT_ON_FAILURE=1 &&
     make format && git diff --exit-code &&
     make doxygen &&
+    if [[ -f warnings.log ]]; then cat warnings.log; test -s warnings.log && exit 1; fi && 
```

Comment:
To @dalg24 other comment, we could just use `DOXYGEN_WARN_AS_ERROR=YES` and drop the whole logic here.

Reply:
That was my intent